rpcserver.go

```diff
@@ -6055,8 +6055,9 @@ func (r *rpcServer) AddInvoice(ctx context.Context,
 				// understand the new BOLT 11 tagged field
 				// containing the blinded path, so we switch
 				// the bit to required.
-				v.Unset(lnwire.Bolt11BlindedPathsOptional)
-				v.Set(lnwire.Bolt11BlindedPathsRequired)
+				v = feature.SetBit(
```

Comment:
Nice, I wonder this is the only case in the codebase where we would switch the dependency to required or vice versa ?

Reply:
I think it might be yeah - but I think we will have more of this in future so I think it is useful to have a helper to depend on instead of needing to manually do this each time